File: modul 4/soal 4/PRAK304-2210817220020-Helena Rosalina.py
a=int(input())

if a>0 and a<10 :
        print("satuan")
    
elif a==0 :
        print("nol")
    
elif a>10 and a<20:
        print("belasan")
    
elif a>19 and a<100 or a==10:
        print("puluhan")

else:
        print("Anda Menginput Melebihi Limit Bilangan")

File: modul 4/soal 4/PRAK304-2210817220020-Helena Rosalina.c
#include <stdio.h>

int main()
{

int a;
    printf(" ");
    scanf("%d", &a);

    if ( a>0 && a<10) {
        printf("satuan");
    }
    else if ( a==0){
        printf("nol");
    }
    else if ( a>10 && a<20){
        printf("belasan");
    }
    else if (a>19 && a<100 || a==10){
        printf("puluhan");
    }
    else {
        printf("Anda Menginput Melebihi Limit Bilangan");
    }

    return 0;
}
